available_images.md

```diff
@@ -84,6 +84,11 @@ You can pin your version by adding the version tag to your URL as follows:
 
      763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:2.4.1-cpu-py37-ubuntu18.04-v1.0
 
+***Note: Starting with MXNet 1.9, PyTorch 1.10, and TensorFlow 2.7, our general framework containers (cpu training, cpu inference, gpu training, gpu inference) 
+are released as "E3" and "SageMaker" images. E3 images support EC2, ECS, and EKS training and inference workflows, whereas SageMaker images
```

Comment:
Same note here -> If they will always be separate releases, should be "released either as E3 or SageMaker"